<launch>

<arg name="rgb_image" value="camera/rgb/image_color"/>
<arg name="depth_image" value="camera/depth/image"/>

<node name="draw_frames" pkg="pips" type="draw_frames" args="/base_link" output="screen">
    <remap from="rgb_image" to="$(arg rgb_image)" />
    <remap from="depth_image" to="$(arg depth_image)" />
  </node>

<node name="play_bag" pkg="rosbag" type="play" args="/media/justin/data/bag_files/tf_fixed/rgb_down_the_hall_2015-10-20-20-56-35.bag --clock --pause -s 13 -r 1" />

</launch>
